### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by IrynaBrenko**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    """
+    """`
```
Comment (Dimosphen1) [inline app/main.py:2]: There is no need to add this change here, remove it, please

File: app/main.py [hunk 2]
```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+    return counter
```
Comment (Dimosphen1) [inline app/main.py:21]: Try to implement solution in one line using built-in string methods and chaining

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested

File: app/main.py (hunk 1 above)
Comment (IrynaBrenko) [reply app/main.py:2]: I don`t underdtand, what change?

**Pull request closed without merging**